build.gradle

```
@@ -39,14 +44,35 @@ apply from: scriptsLocation + 'jacoco.gradle' // jacoco java code coverage
 apply from: scriptsLocation + 'sonarqube.gradle'
 apply from: scriptsLocation + 'vcs.gradle'
 apply from: scriptsLocation + 'semVer.gradle'
+apply from: scriptsLocation + 'tscfg.gradle'
 
 repositories {
   mavenCentral() //searches in bintray's repository 'jCenter', which contains Maven Central
   maven { url 'https://www.jitpack.io' } // allows github repos as dependencies
 }
 
 dependencies {
-  // todo CLEANUP old dependencies
+
+  // scala core
+  implementation "org.scala-lang:scala3-library_3:$scalaVersion"
+  implementation group: 'org.scala-lang.modules', name: 'scala-parallel-collections_3', version: '1.0.4'
```

```suggestion
  implementation 'org.scala-lang.modules:scala-parallel-collections_3:1.0.4'
```

Don't like this kind of notation. :-D